app/javascript/components/collections/landing/FacetBadges.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+
+class FacetBadges extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  handleClick = (index, event) => {
+    event.preventDefault();
+    console.log(index);
+    let newAppliedFacets = this.props.facets.slice();
+    console.log(newAppliedFacets);
+    newAppliedFacets.splice(index, 1);
+    console.log(newAppliedFacets);
+    this.props.search.setState({ appliedFacets: newAppliedFacets });
+  };
+
+  render() {
+    if (this.props.facets != []) {
+      return (
+        <div className="mb-3 facet-badges">
+          {this.props.facets.map((facet, index) => {
+            if (facet.length === 0) {
+              return <div></div>;
```

Comment:
Let's refactor this `return` block to get rid of embedded `returns`.  Line 24 and 26.   And with that cleanup, let's lose the empty `div` return at the bottom of the block as well.